### PR TITLE
[GITHUB-3] Fixes references to undefined variables

### PR DESCRIPTION
**Pull request by wojtek-oledzki**

### CODE REVIEW

File: tasks/configure/assigned_volume.yml [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
   ec2_vol:
-    device_name: "{{ sansible_persistent_volumes_device.device }}"
+    device_name: "{{ sansible_persistent_volumes_device.path }}"
     id: "{{ assigned_volume_ec2_instance_tags.tags[sansible_persistent_volumes_aws.assigned_volume_instance_tag] }}"
     instance: "{{ ansible_ec2_instance_id }}"
     region: "{{ ansible_ec2_placement_region }}"
   when: assigned_volume_mount_facts.volumes[0] is not defined
   register: assigned_volume_connect
   until: assigned_volume_connect is succeeded
-  retries: 6
```
Comment (racer1680) [inline tasks/configure/assigned_volume.yml:36]: What difference does it make?

File: tasks/configure/tagged_volume.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@
   when: tagged_volume_mount_facts.volumes[0] is not defined
   until: tagged_volume_facts.volumes[0].id is defined
   retries: 6
```
Comment (racer1680) [inline tasks/configure/tagged_volume.yml]: You don't want to control retries here? :)

Comment (wojtek-oledzki) [reply tasks/configure/tagged_volume.yml]: missed that one, adding

**Comment by racer1680** [comment]
To resolve "this issue" - I think I know what it is - I added a script in init to force release of attached disk as soon as possible @wojtek-oledzki

**Comment by wojtek-oledzki** [comment]
which is fine, but I still want to have control over how long do I wait for the disk to become available. 

In some instances shutdown might take longer (for whatever reasons)

**Comment by racer1680** [comment]
otherwise... :) LGTM

**Review by racer1680 — APPROVED** [review]
LGTM

**Pull request merged**